shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/common/extractor/TableExtractor.java

```diff
@@ -260,7 +260,7 @@ public void extractTablesFromUpdate(final UpdateStatement updateStatement) {
      */
     public boolean needRewrite(final OwnerSegment owner) {
         for (TableSegment each : tableContext) {
-            if (owner.getIdentifier().getValue().equals(each.getAlias().orElse(null))) {
+            if (owner.getIdentifier().getValue().equalsIgnoreCase(each.getAlias().orElse(null))) {
```

Comment:
Is this sql can be execute in different database. like postgres or oracle?

Reply:
this sql follow sql92 protocol